adapter/settlement.go

```diff
@@ -23,3 +24,63 @@ func (api *Settlement) CreateInstantWalletSettlement(ctx context.Context, reques
 
 	return response.Data, nil
 }
+
+func (api *Settlement) CreatePayoutAccount(ctx context.Context, request CreatePayoutAccountRequest) (*PayoutAccountResponse, error) {
```

Comment:
you can rename CreatePayoutAccountRequest -> CreatePayoutAccountRequestBody

Reply:
We have a naming convention for request types, so it should end with Request